### PR TITLE
Fixed broken CVE2.0 url links in property and markdown files

### PR DESCRIPTION
**Pull request by 0xRCE**
## Description of Change

Changed the urls for CVE 2.0 original and modified entries to the new addres

## Have test cases been added to cover the new functionality?

no, existing are already testing this functionalitiy

### CODE REVIEW

**Comment by jeremylong** [comment]
Can one of the admins verify this patch?

**Review by wakingrufus — APPROVED** [review]

**Comment by baffles** [comment]
`cveUrl20Base` in `cli/src/site/markdown/arguments.md` is incorrect

**Review by acooley — COMMENTED** [review]
Please merge this ASAP.  This is a popular bit of opensource software and builds are breaking because of this.

**Comment by zachofalltrades** [comment]
It will be a whole lot easier for me to fix all of my broken builds by bumping up the version of this plugin rather than adding the additional configuration for the various feed URLs. But I do appreciate the work you are doing with this!

**Review by recena — APPROVED** [review]

File: cli/src/site/markdown/arguments.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Advanced Options
...
 -------|------------------------|-----------------|----------------------------------------------------------------------------------|-------------------
        | \-\-cveUrl12Modified   | \<url\>         | URL for the modified CVE 1.2                                                     | https://nvd.nist.gov/download/nvdcve-Modified.xml.gz
-       | \-\-cveUrl20Modified   | \<url\>         | URL for the modified CVE 2.0                                                     | https://nvd.nist.gov/feeds/xml/cve/nvdcve-2.0-Modified.xml.gz
+       | \-\-cveUrl20Modified   | \<url\>         | URL for the modified CVE 2.0                                                     | https://nvd.nist.gov/feeds/xml/cve/2.0/nvdcve-2.0-Modified.xml.gz
        | \-\-cveUrl12Base       | \<url\>         | Base URL for each year's CVE 1.2, the %d will be replaced with the year          | https://nvd.nist.gov/download/nvdcve-%d.xml.gz
        | \-\-cveUrl20Base       | \<url\>         | Base URL for each year's CVE 2.0, the %d will be replaced with the year          | https://nvd.nist.gov/feeds/xml/cve/nvdcve-2.0-%d.xml.gz
```
Comment (recena) [inline cli/src/site/markdown/arguments.md]: It needs to be changed as well.

**Comment by AndiDog** [comment]
Didn't the 1.2 URLs also change?

**Review by recena — APPROVED** [review]

**Comment by jeremylong** [comment]
Sorry everyone - of course something happens like this when I am traveling for vacation...  I'll get this pushed shortly.

**Pull request merged**